.pre-commit-config.yaml

```diff
@@ -61,6 +61,13 @@ repos:
   - repo: meta
     hooks:
       - id: check-useless-excludes
+  - repo: local
+    hooks:
+      - id: check-json-indentation
+        name: Check JSON Indentation Consistency
+        entry: pre-commit/check_json_indentation.py
```

Comment:
If there isn't an existing hook for this, it would be simpler to use an inline pygrep hook.